types/index.d.ts

```diff
@@ -4,6 +4,12 @@ import { ParserOptions } from 'htmlparser2';
 declare namespace cheerio {
   type AttrFunction = (el: Element, i: number, currentValue: string) => any;
 
+  /** A Selector is used to select DOM elements from document. */
+  type selectorString = string;
```

Comment:
If we adopt this pattern, we should use it for all places that accept either. I do think it is a nice idea, but perhaps as a separate step?

Reply:
True, types `htmlString` and `selectorString` should give you some information what input to expect and `Selector` is already used as interface.